pkg/cli/prompt/text/text_test.go

```diff
@@ -106,7 +106,7 @@ func Test_E2E(t *testing.T) {
 
 		expected := "test prompt\n" +
 			"\n" +
-			"> test placeholder\n" +
+			"> test placeholder                         \n" +
```

Comment:
There was an update to **bubbletea** package which made some of the unit tests in this file to fail. There was an update to the width and the way width is presented. I will be looking into this a little bit more.

Reply:
Do you have a conclusion from looking at this?